specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -169,6 +247,51 @@ model ImageCategoriesAnalysis {
   severity?: int32;
 }
 
+@added(ContentSafety.Versions.v2024_01_30_Preview)
+@doc("The image batch analysis request.")
+model BatchAnalyzeImagesOptions {
+  @doc("The URL of the Azure Storage Blob containing all the images to be analyzed in the batch task.")
+  imagesBlobFolderUrl: url;
+
+  @doc("The URL of the Azure Storage Blob where the batch task results will be written.")
+  analysisResultsBlobFolderUrl: url;
+
+  @doc("The storage mode for the batch task results, either 'CollectiveResultFile' or 'IndividualResultFiles'.")
+  resultsStorageMode?: BatchResultsStorageMode = BatchResultsStorageMode.CollectiveResultFile;
+
+  @doc("The categories will be analyzed. If they are not assigned, a default set of analysis results for the categories will be returned.")
+  categories?: ImageCategory[];
+
+  @doc("This refers to the type of image analysis output. If no value is assigned, the default value will be \"FourSeverityLevels\".")
+  outputType?: AnalyzeImageOutputType = AnalyzeImageOutputType.FourSeverityLevels;
+
+  @doc("The incidents to detect.")
+  incidents?: IncidentOptions;
+}
+
+@added(ContentSafety.Versions.v2024_01_30_Preview)
+@doc("Image batch analyze task.")
+@resource("image/batchAnalyzeTasks")
+model ImageBatchTaskDetail {
+  @doc("The id of image batch analysis task.")
+  @key("taskId")
+  @visibility("read")
+  @maxLength(64)
+  taskId: string;
+
+  @doc("The status of the batch image analysis task.")
+  status: BatchTaskStatus;
+
+  @doc("The progress of the batch image analysis task, represented as a percentage (0-100).")
```

Comment:
Nit: It's atypical to have a float as a percentage from values 0..100. It's _far_ more common for 0.0..1.0. In that regard, this would probably just be called `progress`.

Reply:
Make sense!
Moved custom category to next version.

Reply:
We will use int instead of float. But I think the parameter name should still contain "percentage". This clearly shows what the parameter means.